packages/flanc/src/graceful-shutdown.ts

```diff
@@ -15,6 +15,7 @@ function shutdown(server: AppServer, onShutdown?: (server: AppServer) => any) {
     server.ready = false;
     if (onShutdown) onShutdown(server);
     setTimeout(server.stop, shutdownRoutine.serverClose);
+    setTimeout(process.exit(0), shutdownRoutine.serverClose + 1000);
```

Comment:
@dylang93 I've been looking at the docs for a while- technically `stoppable` should be in charge of of shutting everything down when the httpServer stops. 

If you have an issue still with your graceful shutdown, make sure that the `shutdownRoutine.appKill` is greater than your `shutdownRoutine.serverClose` and the result of `shutdownRoutine.appKill - shutdownRoutine.serverClose` is greater than 0.

If there's still an issue, it might be a problem with your `stoppable` version.

Though I think this PR highlights a clear issue: we should always enforce the grace value to be higher than 0 , and we could probably simplify the config to just one value.

Ex: `shutdownRoutine.serverGracePeriod` would be the total grace time, or the current appKill value, and the serverClose should be 50% of that.

What do you think ?